From 3106c2b2421a190dd3eff4a1e37294e5f9a963e5 Mon Sep 17 00:00:00 2001
From: Adrian Bridgett <bridgett@debian.org>
Date: Mon, 30 Oct 2017 00:41:40 +0200
Subject: Rename some files - this is Dante, not the original SOCKS.

- /etc/socks.conf => /etc/dante.conf
- /etc/sockd.conf => /etc/danted.conf
- /usr/sbin/sockd => /usr/sbin/danted

The changelog entry references bug #35187, but bugs.debian.org disclaims any
knowledge of it.

Forwarded: not-needed
Last-Update: 2010-07-01
Patch-Name: rename-programs.patch
---
 INSTALL          |  4 ++--
 doc/sockd.8      | 14 +++++++-------
 doc/sockd.conf.5 | 24 ++++++++++++------------
 doc/socks.conf.5 |  8 ++++----
 4 files changed, 25 insertions(+), 25 deletions(-)

diff --git a/INSTALL b/INSTALL
index 1545f2e..d5de432 100644
--- a/INSTALL
+++ b/INSTALL
@@ -381,8 +381,8 @@ RUNTIME CONFIGURATION FILES
 
 No configuration-files are installed, these must be installed
 manually.
-The Dante server expects it's configuration file to be in /etc/sockd.conf.
-The client library uses the file /etc/socks.conf.
+The Dante server expects it's configuration file to be in /etc/danted.conf.
+The client library uses the file /etc/dante.conf.
 
 The file bin/socksify.sh contains a shell script that allows one
 to dynamically socksify a dynamically linked application at runtime.
diff --git a/doc/sockd.8 b/doc/sockd.8
index e49a50d..e8a8889 100644
--- a/doc/sockd.8
+++ b/doc/sockd.8
@@ -41,11 +41,11 @@
 .\" any improvements or extensions that they make and grant Inferno Nettverk A/S
 .\" the rights to redistribute these changes.
 .\"
-.TH SOCKD 8 "July 29 2013"
+.TH DANTED 8 "July 29 2013"
 .SH NAME
-sockd \- network proxyserver
+danted \- network proxyserver
 .SH SYNOPSIS
-.B sockd
+.B danted
 .RB [ \-DLVdhnv ]
 .RB [ \-N
 .IR number ]
@@ -103,7 +103,7 @@ Disables TCP keep-alive messages.  Normally
 enables TCP keep-alive messages so that connections from machines that
 have crashed or for other reasons no longer can be reached time out.
 Note that this can take several hours.  See also the "timeout"
-option in sockd.conf(5).
+option in danted.conf(5).
 .TP
 .BI \-p file
 .B Dante
@@ -134,10 +134,10 @@ It is recommended that
 be set to a (local) filesystem with low latency.  The directory
 should have read/write/execute permission by both the
 privileged and unprivileged userid (as specified in
-/etc/sockd.conf), but no other non-privileged users.
+/etc/danted.conf), but no other non-privileged users.
 .SH FILES
 .TP
-/etc/sockd.conf
+/etc/danted.conf
 .B Dante
 server configuration file.
 .SH AUTHORS
@@ -145,7 +145,7 @@ For inferno Nettverk A/S:
    Michael Shuldman
    Karl-Andre' Skevik
 .SH SEE ALSO
-socks.conf(5), sockd.conf(5)
+dante.conf(5), danted.conf(5)
 .PP
 Information about new releases and other related issues can be found
 on the \fBDante\fP WWW home page: http://www.inet.no/dante/
diff --git a/doc/sockd.conf.5 b/doc/sockd.conf.5
index 3e0d4b1..d1f8f7d 100644
--- a/doc/sockd.conf.5
+++ b/doc/sockd.conf.5
@@ -41,9 +41,9 @@
 .\" any improvements or extensions that they make and grant Inferno Nettverk A/S
 .\" the rights to redistribute these changes.
 .\"
-.TH SOCKD.CONF 5 "July 29 2013"
+.TH DANTED.CONF 5 "July 29 2013"
 .SH NAME
-sockd.conf \- \fBDante\fP server configuration file syntax
+danted.conf \- \fBDante\fP server configuration file syntax
 .SH DESCRIPTION
 The configuration file for the \fBDante\fP server controls both access
 controls and logging.  It is divided into three parts; server settings,
@@ -315,7 +315,7 @@ Setting it to 0 will use the systems default.
 .IP \fBtimeout.io\fP
 The number of seconds an established connection can be idle.
 The default is 0, meaning forever.
-See also the "-n" option in the sockd(8) manpage.
+See also the "-n" option in the danted(8) manpage.
 
 Individual timeouts can be set for TCP and UDP by suffixing io with
 ".<protocolname>", i.e. \fBtimeout.io.tcp\fP or \fBtimeout.io.udp\fP.
@@ -353,8 +353,8 @@ values.  The \fBDante\fP server can use two different userids, or three
 if compiled with libwrap support.  They are as follows:
 .IP \fBuser.privileged\fP
 Username which will be used for doing privileged operations.
-If you need special privileges to read the sockd.conf file or to write
-the sockd.pid file (you can create it manually before starting sockd),
+If you need special privileges to read the danted.conf file or to write
+the danted.pid file (you can create it manually before starting danted),
 have anything in your configuration that requires binding privileged
 TCP/UDP ports (ports below 1024), or use some sort of password-based
 authentication, this probably needs to be set to root.
@@ -676,7 +676,7 @@ Possible values are \fBclear\fP, \fBintegrity\fP, or \fBconfidentiality\fP.
 The default is to accept whatever the client offers
 except \fBclear\fP, as \fBclear\fP is not part of the SOCKS GSSAPI standard.
 .IP \fBgssapi.keytab\fP
-Value for keytab to use.  The default is "FILE:/etc/sockd.keytab".
+Value for keytab to use.  The default is "FILE:/etc/danted.keytab".
 .IP \fBgssapi.servicename\fP
 Which servicename to use when involving GSSAPI.  Default is "rcmd".
 
@@ -693,7 +693,7 @@ below for details.
 Require that the connection be "authenticated" using one of the
 given clientmethods.
 .IP \fBpam.servicename\fP
-Which servicename to use when involving pam.  Default is "sockd".
+Which servicename to use when involving pam.  Default is "danted".
 .IP \fBport\fP
 Parameter to \fBfrom\fP, \fBto\fP and \fBvia\fP.  Accepts the keywords
 \fBeq/=, neq/!=, ge/>=, le/<=, gt/>, lt/<\fP followed by a number.
@@ -772,7 +772,7 @@ the target destination (see \fBcheckreplyauth\fP for information about
 authentication the target destination).  Valid values are the same as in
 the global \fBsocksmethod\fP line.
 .IP \fBpam.servicename\fP
-What servicename to use when involving pam.  Default is "sockd".
+What servicename to use when involving pam.  Default is "danted".
 
 .IP \fBport\fP
 Parameter to \fBfrom\fP, \fBto\fP and \fBvia\fP.  Accepts the keywords
@@ -1054,12 +1054,12 @@ multiple alarms are specified in a monitor rule.
 .SH ROUTES
 The routes are specified with a \fBroute\fP keyword.  Inside a pair of
 curly braces ({}) a set of keywords control the behavior of the route.
-See socks.conf(5) for a description.  This is used to perform so-called
+See dante.conf(5) for a description.  This is used to perform so-called
 "server-chaining", where one socks-server connects to another socks-server
 further upstream.
 
 The syntax for these routes is the same as the routes used by the client.
-Please see socks.conf(5) for information about the route syntax.
+Please see dante.conf(5) for information about the route syntax.
 
 There are however some special things one need to be aware of
 regarding serverchaining and routes specified for the server:
@@ -1081,7 +1081,7 @@ See the example/ directory in the distribution.
 .SH FILES
 .nf
 .ta \w 1
-/etc/sockd.conf   \fBDante\fP server configuration file.
+/etc/danted.conf   \fBDante\fP server configuration file.
 /etc/passwd       systemfile used when doing standard username/password
                   authentication.
 .fi
@@ -1090,7 +1090,7 @@ For inferno Nettverk A/S:
    Michael Shuldman
    Karl-Andre' Skevik
 .SH SEE ALSO
-sockd(8), socks.conf(5), hosts_access(5)
+danted(8), dante.conf(5), hosts_access(5)
 
 .PP
 Information about new releases and other related issues can be found
diff --git a/doc/socks.conf.5 b/doc/socks.conf.5
index 38be0fd..daee61f 100755
--- a/doc/socks.conf.5
+++ b/doc/socks.conf.5
@@ -41,9 +41,9 @@
 .\" any improvements or extensions that they make and grant Inferno Nettverk A/S
 .\" the rights to redistribute these changes.
 .\"
-.TH SOCKS.CONF 5 "July 29 2013"
+.TH DANTE.CONF 5 "July 29 2013"
 .SH NAME
-socks.conf \- socks client configuration file syntax
+dante.conf \- socks client configuration file syntax
 .SH DESCRIPTION
 The configuration file for the socks client library allow control over
 logging and server selection.  It is divided into two parts;
@@ -186,13 +186,13 @@ whether the application to socksify needs threadlocking or not.
 Some additional environment variables are documented in socksify(1).
 .TP
 .SH FILES
-.I /etc/socks.conf
+.I /etc/dante.conf
 .SH AUTHORS
 For inferno Nettverk A/S:
    Michael Shuldman
    Karl-Andre' Skevik
 .SH SEE ALSO
-socksify(1), sockd(8), sockd.conf(5)
+socksify(1), danted(8), danted.conf(5)
 
 .PP
 Information about new releases and other related issues can be found on
